src/FlipMove.js

```diff
@@ -119,21 +94,21 @@ class FlipMove extends Component {
     // longer, so that we can animate them away.
     this.setState({
       children: this.isAnimationDisabled(nextProps)
-        ? nextChildren
-        : this.calculateNextSetOfChildren(nextChildren)
+        ? nextChildren.map(element => ({ ...element, element }))
```

Comment:
Curious why this was changed? I know earlier you spoke about keeping this type of object:

```js
{ element: ReactElement, appearing: boolean, leaving: boolean, entering: boolean }
```

The way this is now, though, we wind up nesting the element within itself:

```js
{
  element: ReactElement,
  ...ReactElement,
  appearing: boolean,
  leaving: boolean,
  entering: boolean,
}
```

Curious if this is intentional, or a "leftover" from the earlier change.